scripts/base/frameworks/intel/main.zeek

```diff
@@ -30,7 +30,8 @@ export {
 		USER_NAME,
 		## Certificate SHA-1 hash.
 		CERT_HASH,
-		## Public key MD5 hash. (SSH server host keys are a good example.)
+		## Public key MD5 hash, formatted as hexadecimal digits delimited by colons.
+		## (SSH server host keys are a good example.)
```

Comment:
I am honestly not quite sure how I feel about this. You are right that for the public keys in SSH connections, it has been common to include colons between the hex digits.

This, however, does not seem to be the case for public key hashes when used in other circumstances (like, e.g., in certificates). So - it basically feels like by doing this we are painting ourselves into a corner where PUBKEY_HASH actually means SSH_PUBKEY_MD5_HASH.

That being said, the naming of the other intel items also does not seem do be optimal, so this might not matter super much.

Reply:
Thanks, good point.  I could go either way since at the end of the day it's not a hard corner to escape from since it's "just" a string-formatting difference (add/remove colons) and also thinking it's conceptually used like a "fingerprint" matching system with an SSH fingerprint having traditional format and also the only current usage, though doesn't have to be exclusive to SSH (but also agree about the actual names of Intel items being not-so-precise).